doc/source/tech_note/Vegetation_Phenology_Turnover/CLM50_Tech_Note_Vegetation_Phenology_Turnover.rst

```diff
@@ -5,8 +5,8 @@ Vegetation Phenology and Turnover
 
 The CLM phenology model consists of several algorithms controlling the
 transfer of stored carbon and nitrogen out of storage pools for the
-display of new growth and into litter pools for losses of displayed
-growth. PFTs are classified into three distinct phenological types that
+display of new growth, into litter pools for losses of displayed
```

Comment:
I realize that trees can also be used for biofuels, but I’m not sure the new code is set up to handle this (I think the biofuels code is within the crop itv switch throughout the code). It is therefore perhaps a little confusing to include reference to biofuels in the phenology chapter since phenology for other crops are not mentioned in this chapter, and it differs from the natural PFT phenology described here. I suggest removing the reference to biofuels in this section, particularly because the text here is very similar to what is already included in the crop chapter. We can add text here when we can allow for the biofuel functionality to be used for non-crop PFTs.